### ISSUE TITLE
It is not possible to configure pool

### ISSUE THREAD

**Issue by dizonn** [jeetmp3/spring-session#8]
If sentinel master and nodes are not specified than redisConnectionFactory is configured without poolConfig

**Comment by jeetmp3** [comment]
Hi @dizonn by default redisConnectionFactory use pool, if you don't want to use pool then set the property `springsession.redis.connectionFactory.usePool = false`. If you want to override poolConfig then create a JedisPoolConfig bean with name `poolConfig` in your config.
Hope this answer your question.

**Comment by dizonn** [comment]
redisConnectionFactory  does not use overridden pool since it is declared without reference to it.
To fix this you can declare factory this way:
`redisConnectionFactory(JedisConnectionFactory, ref("poolConfig"))`

or

```
redisConnectionFactory(JedisConnectionFactory) {
hostName = conf.redis.connectionFactory.hostName ?: "localhost"
...
poolConfig = ref('poolConfig')
}
```

**Comment by jitendra-bisht** [comment]
@dizonn Yes same is configured in 1.2.1 branch and available in plugin version 1.2.1
Thanks to let us know about the issue.
